nntrainer/src/tensor.cpp

```diff
@@ -758,6 +758,14 @@ std::ostream &operator<<(std::ostream &out, Tensor const &m) {
   return out;
 }
 
+float *Tensor::getAddress(unsigned int i) {
+  if (i > this->dim.getDataLen()) {
+    ml_loge("Error: Index out of bounds");
+    return &data[0];
```

Comment:
IMO, this case should return NULL or equivalent if there isn't specific use case.
```suggestion
    return nullptr;
```

Reply:
right. it's better.